po/tr.po

```diff
@@ -335,707 +329,708 @@ msgstr "Fonksiyon Çağrılarında Ömürler"
-#: src/SUMMARY.md:255
+#: src/SUMMARY.md:253
 msgid "Useful Crates"
-msgstr ""
+msgstr "Kullanışlı Sandıklar (Crates)"
 
-#: src/SUMMARY.md:256
+#: src/SUMMARY.md:254
 msgid "zerocopy"
-msgstr ""
+msgstr "zerocopy"
 
-#: src/SUMMARY.md:257
+#: src/SUMMARY.md:255
 msgid "aarch64-paging"
-msgstr ""
+msgstr "aarch64-paging"
 
-#: src/SUMMARY.md:258
+#: src/SUMMARY.md:256
 msgid "buddy_system_allocator"
-msgstr ""
+msgstr "buddy_system_allocator"
 
-#: src/SUMMARY.md:259
+#: src/SUMMARY.md:257
 msgid "tinyvec"
-msgstr ""
+msgstr "tinyvec"
 
-#: src/SUMMARY.md:260
+#: src/SUMMARY.md:258
 msgid "spin"
-msgstr ""
+msgstr "spin"
 
-#: src/SUMMARY.md:262 src/bare-metal/android/vmbase.md:1
+#: src/SUMMARY.md:260 src/bare-metal/android/vmbase.md:1
 msgid "vmbase"
-msgstr ""
+msgstr "vmbase"
 
-#: src/SUMMARY.md:264
+#: src/SUMMARY.md:262
 msgid "RTC Driver"
-msgstr ""
+msgstr "RTC Sürücüsü"
 
-#: src/SUMMARY.md:267
+#: src/SUMMARY.md:265
 msgid "Concurrency: Morning"
-msgstr ""
+msgstr "Eşzamanlılık: Sabah"
 
-#: src/SUMMARY.md:272 src/concurrency/threads.md:1
+#: src/SUMMARY.md:270 src/concurrency/threads.md:1
 msgid "Threads"
-msgstr ""
+msgstr "İş Parçacıkları (Threads)"
 
-#: src/SUMMARY.md:273 src/concurrency/scoped-threads.md:1
+#: src/SUMMARY.md:271 src/concurrency/scoped-threads.md:1
 msgid "Scoped Threads"
-msgstr ""
+msgstr "Kapsamlı İş Parçacıkları (Threads)"
 
-#: src/SUMMARY.md:274 src/concurrency/channels.md:1
+#: src/SUMMARY.md:272 src/concurrency/channels.md:1
 msgid "Channels"
-msgstr ""
+msgstr "Kanallar"
 
-#: src/SUMMARY.md:275 src/concurrency/channels/unbounded.md:1
+#: src/SUMMARY.md:273 src/concurrency/channels/unbounded.md:1
 msgid "Unbounded Channels"
-msgstr ""
+msgstr "Sınırsız Kanallar"
 
-#: src/SUMMARY.md:276 src/concurrency/channels/bounded.md:1
+#: src/SUMMARY.md:274 src/concurrency/channels/bounded.md:1
 msgid "Bounded Channels"
-msgstr ""
+msgstr "Sınırlı Kanallar"
 
-#: src/SUMMARY.md:277
+#: src/SUMMARY.md:275
 msgid "Send and Sync"
-msgstr ""
+msgstr "Send ve Sync"
 
-#: src/SUMMARY.md:277
+#: src/SUMMARY.md:275
 msgid "Send"
-msgstr ""
+msgstr "Send"
 
-#: src/SUMMARY.md:277
+#: src/SUMMARY.md:275
 msgid "Sync"
-msgstr ""
+msgstr "Sync"
 
-#: src/SUMMARY.md:280 src/concurrency/send-sync/examples.md:1
+#: src/SUMMARY.md:278 src/concurrency/send-sync/examples.md:1
 msgid "Examples"
-msgstr ""
+msgstr "Örnekler"
 
-#: src/SUMMARY.md:281 src/concurrency/shared_state.md:1
+#: src/SUMMARY.md:279 src/concurrency/shared_state.md:1
 msgid "Shared State"
-msgstr ""
+msgstr "Paylaşımlı Durum (State)"
 
-#: src/SUMMARY.md:282
+#: src/SUMMARY.md:280
 msgid "Arc"
-msgstr ""
+msgstr "Arc"
 
-#: src/SUMMARY.md:283
+#: src/SUMMARY.md:281
 msgid "Mutex"
-msgstr ""
+msgstr "Mutex"
 
-#: src/SUMMARY.md:286 src/SUMMARY.md:307
+#: src/SUMMARY.md:284 src/SUMMARY.md:305
 #: src/exercises/concurrency/dining-philosophers.md:1
 #: src/exercises/concurrency/solutions-morning.md:3
 msgid "Dining Philosophers"
-msgstr ""
+msgstr "Yemek Filozofları"
```

Comment:
Hello, the same problem has been referred [here](https://bilgisayarkavramlari.com/2012/01/22/filozoflarin-aksam-yemegi-dining-philosophers/) as Filozofların Akşam Yemeği. Do you think it may be more helpful to the reader if we use that term? Thanks

Reply:
I was wrong here. I thought it was an example name. So I assumed that direct translation wouldn't matter. But the name in this title is a known problem. Can I use the below in Turkish?

https://tr.wikipedia.org/wiki/Makarna_yiyen_d%C3%BC%C5%9F%C3%BCn%C3%BCrler_sorunu

Reply:
I choose your "Filozofların Akşam Yemeği".